src/Sculpin/Bundle/MarkdownBundle/CommonMarkParser.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Sculpin\Bundle\MarkdownBundle;
+
+use League\CommonMark\CommonMarkConverter;
+use Sculpin\Core\Converter\ParserInterface;
+
+/**
+ * Class CommonMarkParser
+ */
+class CommonMarkParser extends CommonMarkConverter implements ParserInterface
```

Comment:
I wanted to stay consistent with how the other converters are put together.  No other reason.